### PR TITLE
Allow to  specify user or org

### PR DESCRIPTION
**Pull request by raphink**

### CODE REVIEW

File: cmd/fork-cleaner/main.go [hunk 1]
```diff
@@ -35,6 +35,10 @@ func main() {
 			Usage:  "Base GitHub URL",
 			Value:  "https://api.github.com/",
 		},
+		cli.StringFlag{
+			Name:  "user, u",
```
Comment (caarlos0) [inline cmd/fork-cleaner/main.go:39]: maybe `handle` instead?

Comment (raphink) [reply cmd/fork-cleaner/main.go:39]: If you prefer

Comment (raphink) [reply cmd/fork-cleaner/main.go:39]: Short option would be `h` though...

Comment (caarlos0) [reply cmd/fork-cleaner/main.go:39]: h would conflict with help...

Comment (raphink) [reply cmd/fork-cleaner/main.go:39]: So what do you suggest? I stay with `user`?

**Pull request closed without merging**